geniza/corpus/tests/test_corpus_templates.py

```diff
@@ -58,6 +59,18 @@ def test_multi_viewer(self, client, join):
         second_url = join.textblock_set.last().fragment.iiif_url
         assertContains(response, f'data-iiif-urls="{first_url} {second_url}"')
 
+    def test_edit_link(self, client, document):
```

Comment:
pytest-django has an `admin_client` fixture https://pytest-django.readthedocs.io/en/latest/helpers.html#admin-client-django-test-client-logged-in-as-admin 

let's use that here — although it doesn't test the exact permission, that seems fine